### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by rootDEV2990**
This branch contains our Images and Video embed project. Validator confirmed and README file updated. Click user icon on top right for switch between dark and light mode.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -22,6 +76,13 @@ nav .logo img {
 	width: 100px;
 	padding-left: 15%;
 }
+nav .menuicon {
+	height: 80%;
+	width: 45px!important;
```
Comment (pelzolga123) [inline css/style.css]: `!important` is a bad practice and should not be used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: css/style.css [hunk 2]
```diff
@@ -161,20 +274,151 @@ font-size:16px;
...
+	height:50px;
+}
+.upload-info div img {
+	height: 50px;
+}
+article .pt7 img {
+    height: 40px;
+}
+section iframe {
+	/*padding-top: 50px;*/
```
Comment (pelzolga123) [inline css/style.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: great!

Here are the things you need to change, before this project can be approved:

* [ ] The search bar is aligned to the middle of the navbar ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/searchbar_align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
* [ ] Project should have like and dislike icons, right now you have images there ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_like_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
* [ ] Project should have a share button, right now it's image ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
* [ ] Nav bar doesn't look like original right now, please try to make search button and hamburger icon image smaller, so that everything can be aligned properly. 

Your navbar:
image

Original:
image


#### [OPTIONAL:]
* [ ] It's a good practice to use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links
* [ ] It's a good practice to use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by majovanilla** [comment]
# Code Review Request under review 👀 by Maria Reyes

**Comment by majovanilla** [comment]
Hey @rootDEV2990  @rootDEV2990 , 👋

## Good job on implementing the required changes on this project! 👍

Just some quick reminders: 

- You don't need to put the nav tag inside a div ;)
- Avoid using repetitive commits like you did on the last part. If you want good commit examples, check here: https://chris.beams.io/posts/git-commit/

**You are good to go! You can merge your branch!** 😀

## PROJECT STATUS: APPROVED

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes



You can find me on Social Media below 👇 . Remember to introduce you and say hi!

[GitHub](https://github.com/majovanilla) [Twitter](https://twitter.com/MajoVanilla) [LinkedIn](https://www.linkedin.com/in/majoreyesparroquin/)

**Pull request merged**